deploy/vagrant-test-install/debian11-mobile/Vagrantfile

```diff
@@ -22,71 +22,73 @@ Vagrant.configure("2") do |config|
   end
 
   config.vm.provision "shell", inline: <<-SHELL
-    set -e
+    #set -e
```

Comment:
if we do this build failures will be reported. If a single line is allowed to fail, can we add `|| true` at the end

Reply:
I changed the box to debian/bullseye64 and readded the exit option. I believe this was because of a problem in the generic/debian11 box